### PR TITLE
improve installation process and update README

### LINKED ISSUE #15: update README and simplify installation
the installation should be simplified like in the server, and the README updated correspondingly

### PR DESCRIPTION
**Pull request by robinwersich**
this should make the bot ready for hand-in

resolves #15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,20 @@
 # gameprog-detective-bot
 
-This is the `bot` part of the Gameprog Project of Antonio Dimeo, Tim Garrels, Paul Methfessel and Robin Wersich.
+Dies ist der `bot` Teil des Gameprog Projektes von Antonio Dimeo, Tim Garrels, Paul Methfessel und Robin Wersich.
+Eine umfassende README ist im repository des [servers](https://github.com/EatingBacon/gameprog-detective-server) zu finden.
```
Comment (EatingBacon) [inline README.md]:
```suggestion
Eine umfassende README ist im repository des [Servers](https://github.com/EatingBacon/gameprog-detective-server) zu finden.
```

File: README.md [hunk 2]
```diff
@@ -1,28 +1,20 @@
...
-source .venv/bin/activate
-```
-
-- Use `./manage.sh install` to install and setup the bot
-- Use `./manage.sh start` to start the bot
-- For more commands use `./manage.sh help`
+Voraussetzung ist ein installiertes Python 3
+1. `./manage.sh install` um den bot zu installieren
+1. `./manage.sh start` um den bot zu starten
+1. für weitere befehle `./manage.sh help` benutzen
```
Comment (EatingBacon) [inline README.md]:
```suggestion
Voraussetzung ist ein installiertes Python 3
1. `./manage.sh install` um den Bot zu installieren
1. `./manage.sh start` um den Bot zu starten
1. für weitere Befehle `./manage.sh help` benutzen
```

File: README.md [hunk 3]
```diff
@@ -1,28 +1,20 @@
...
 ## About
-This bot uses the [python-telegram-bot wrapper](https://python-telegram-bot.org/). `bot.py` registeres a `/start` and a `/describe` command and a `text-handler`:  
-`/start` - Register command provided by the [app](https://github.com/ADimeo/gameprog-detective-app) to associate `telegramHandle` with `userId`  
-`/describe` - Prints description of current [storypoint](https://github.com/EatingBacon/gameprog-detective-server/wiki/Story-Storypoint)  
-`text-handler` - Main part of the bot, used to proceed in the story (main interaction)
-
-`bot/server_interaction.py` wraps important API calls to the [server](https://github.com/EatingBacon/gameprog-detective-server) into function calls.  
-`bot/telegram_interaction.py` handles chat interaction with the player by interpreting and answering messages.
+Dieser Bot benutzt den [python-telegram-bot wrapper](https://python-telegram-bot.org/). `bot.py` registriert einen `/start` und einen `/describe` Befehl und einen `text-handler`:  
+- `/start` - Registrierungs Befehl durch die [App](https://github.com/ADimeo/gameprog-detective-app) um `telegramHandle` mit `userId` zu verbinden
```
Comment (EatingBacon) [inline README.md]:
```suggestion
- `/start` - Registrierungs Befehl seitens der [App](https://github.com/ADimeo/gameprog-detective-app) um `telegramHandle` mit `userId` zu verbinden
```

File: README.md [hunk 4]
```diff
@@ -1,28 +1,20 @@
...
+- `/describe` - Lässt den Bot die Beschreibung des aktuellen [storypoint](https://github.com/EatingBacon/gameprog-detective-server/wiki/Story-Storypoint) senden
```
Comment (EatingBacon) [inline README.md]:
```suggestion
- `/describe` - Lässt den Bot die Beschreibung des aktuellen [Storypoints](https://github.com/EatingBacon/gameprog-detective-server/wiki/Story-Storypoint) senden
```

File: README.md [hunk 5]
```diff
@@ -1,28 +1,20 @@
...
+- `text-handler` - Hauptkomponente des Bots, um die Story weiterzuführen
 
 ## Config
-This bot needs a [server](https://github.com/EatingBacon/gameprog-detective-server). You have to specify the url of that server in the env-var `GP_SERVER_URL`.  
-This bot needs a telegram-bot-token. You have to specify the token in the env-var `GP_TELEGRAM_BOT_TOKEN`. Refer to the official telegram doc to learn [how to request a bot token](
-https://core.telegram.org/bots#6-botfather).
+- Dieser Bot benötigt einen [server](https://github.com/EatingBacon/gameprog-detective-server) um zu funktionieren. Die IP des servers muss während des Installationsprozesses eingegeben werden
```
Comment (EatingBacon) [inline README.md]:
```suggestion
- Dieser Bot benötigt einen [Server](https://github.com/EatingBacon/gameprog-detective-server) als Backend. Die IP des Servers muss während des Installationsprozesses eingegeben werden
```

File: manage.sh [hunk 6]
```diff
@@ -17,23 +17,41 @@ if [ "$command" == "install" ]; then
...
+fi
+
+# try to load venv
+if [ -d .venv ]; then
+    source .venv/bin/activate
+else
+    echo "please first install the bot"
+    exit
+fi
+[ -f env_vars ] && source env_vars
```
Comment (EatingBacon) [inline manage.sh]: Dont these two lines conflict each other?

**Review by timgarrels — CHANGES_REQUESTED** [review]

File: manage.sh (hunk 6 above)
Comment (robinwersich) [reply manage.sh]: no, why would they? The `source` command just executes the given shell script, but not in a subshell as usual but directly in the calling shell. So you can source as many scripts as you want

File: README.md (hunk 1 above)
Comment (EatingBacon) [inline README.md]:
```suggestion
Eine umfassende README ist im Repository des [Servers](https://github.com/EatingBacon/gameprog-detective-server) zu finden.
```

**Review by timgarrels — CHANGES_REQUESTED** [review]

**Pull request merged**